recipes/pcst-fast/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "pcst-fast" %}
+{% set version = "1.0.10" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://files.pythonhosted.org/packages/d4/fd/64b51c867bad63e6622ff97cad6230b94b19b5a61e30424cd69c8353091c/pcst_fast-{{ version }}.tar.gz
```

Current idiom would be:

```suggestion
  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pcst_fast-{{ version }}.tar.gz
```